kube_scheduler/tests/test_sli_metrics.py

```diff
@@ -0,0 +1,87 @@
+# (C) Datadog, Inc. 2023-present
+# All rights reserved
+# Licensed under a 3-clause BSD style license (see LICENSE)
+
+import os
+
+import mock
+import pytest
+import requests
+import requests_mock
+
+from datadog_checks.kube_scheduler import KubeSchedulerCheck
+
+instance = {'prometheus_url': 'http://localhost:10259/metrics'}
+
+# Constants
+CHECK_NAME = 'kube_scheduler'
+NAMESPACE = 'kube_scheduler'
+
+
+@pytest.fixture()
+def mock_metrics():
+    f_name = os.path.join(os.path.dirname(__file__), 'fixtures', 'metrics_slis_1.27.3.txt')
```

Comment:
Instead of `os.path.dirname(__file__)`, you can call `get_here()`. Example: https://github.com/DataDog/integrations-core/blob/35670741201da519a7d69af53822a8d94897183c/kong/tests/common.py#L11